package.json

```diff
@@ -18,7 +18,7 @@
     "intl": "1.2.5",
     "jail-monkey": "2.0.0",
     "jsc-android": "236355.1.1",
-    "mattermost-redux": "github:mattermost/mattermost-redux#16a39c671193db23006d848c7866adbb5cae6901",
+    "mattermost-redux": "github:mattermost/mattermost-redux#47bd050275818dbe3d7866723c6a479b489e3b41",
```

Comment:
Is this reference on master, was the redux PR merged? When cherry-pick do not forget to change the reference to mattermost-redux commit on the release branch